assets/components/playlist.html

```diff
@@ -1,45 +1,54 @@
-<h2 class="ui dividing header">Videos
-  <div class="ui toggle checkbox">
-    <input
-      type="checkbox"
-      ng-model="$ctrl.notifications"
-      ng-change="$ctrl.toggleNotifications($ctrl.notifications)"
-      name="public-chat"
-    >
-    <label>Notifications</label>
+<div class="ui top attached tabular menu">
+  <div
+    ng-click="$ctrl.setTab('up-next')"
+    ng-class="($ctrl.activeTab==='up-next') ? 'active item' : 'item'"
+  >
+    Up Next
+    <div class="ui toggle checkbox">
```

Comment:
Can you add a bit of padding to the left of this? It's very close to the Up Next text.